testsuite/pom.xml

```diff
@@ -36,6 +36,7 @@
         <org.jboss.eap.datasources.mssqlserver.driver.version>9.2.1.jre8</org.jboss.eap.datasources.mssqlserver.driver.version>
         <org.jboss.eap.datasources.oracle.driver.version>23.6.0.24.10</org.jboss.eap.datasources.oracle.driver.version>
         <org.jboss.eap.datasources.postgresql.driver.version>42.2.20</org.jboss.eap.datasources.postgresql.driver.version>
+        <org.jboss.eap.datasources.db2.driver.version>11.5.9.0</org.jboss.eap.datasources.db2.driver.version>
```

Comment:
we should use the most recent version https://repo1.maven.org/maven2/com/ibm/db2/jcc/12.1.2.0/